pilot/pkg/kube/inject/webhook.go

```diff
@@ -241,6 +247,13 @@ func (wh *Webhook) Run(stop <-chan struct{}) {
 	}
 }
 
+// GetCABundlePEM returns the CA bundle PEM, read from the cert file. Safe for concurrent use.
+func (wh *Webhook) GetCABundlePEM() []byte {
+	wh.mu.Lock()
+	defer wh.mu.Unlock()
+	return wh.caCertPEM
+}
+
```

Comment:
I didn't like entangling the patch functionality into this type, so I moved it entirely into main.